share/validate-product-config/checks/check-site-sections.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Make sure all site sections from the document are also configured
+# in the INI file
+
+file=$1
+
+if [[ "$site_sections" ]]; then
+  echo "Checking @site-section for $file with $site_sections"
+  sites=$($starlet sel -t -v "//@site-section" $file | tr " " "\n")
+  site_sections=$(echo "${site_sections}" | tr -s " " "\n")
```

Comment:
Here, `sites` v/ `site_sections` seems rather unclear -- hence the above suggestion to use `valid_site_sections`.

Reply:
Also fixed in commit 11192e9